ui/src/app/components/rollout-actions/rollout-actions.tsx

```diff
@@ -36,6 +37,7 @@ export const RolloutActionButton = (props: {action: RolloutAction; rollout: Roll
                 label: 'RETRY',
                 icon: 'fa-redo-alt',
                 action: api.rolloutServiceRetryRollout,
+                disabled: props.rollout.status !== RolloutStatus.Degraded,
```

Comment:
why is this added? you can retry degraded rollout as well

Reply:
Yes I think the button should only be enabled when the rollout is degraded, right?  The double negative is a little confusing :)

Reply:
i misread the line..yes you are correct